### DISCUSSION TITLE
Child Documents in Quarto

### DISCUSSION THREAD

**Discussion by AliSajid** [quarto-dev/quarto-cli#1535] [Q&A] [answered]
Hi Everyone,

We have a few reports that we generate programmatically with R Markdown. I'm finding Quarto to be particularly useful for that and considering migrating to quarto for the next upgrade to our pipeline. However, One thing that I have implemented is parametrized child documents that are add chunks to the main document.

Is there any plan for quarto to implement or add a functionality like that? Something like `knitr::knit_child()`? Thanks.

**Comment by mcanouil** [comment, marked as answer]
Quarto with knitr engine should (at least that is the case for now in my experience) behave as RMarkdown.
Also, there is a shortcode to include Quarto document (see https://quarto.org/docs/authoring/includes.html).

**Comment by cderv** [comment]
>  However, One thing that I have implemented is parametrized child documents that are add chunks to the main document.

Can you share more on this ? What do you do exactly ? how does your implementation works ? Can you share some repo example ?
